ui/plugin-admin/src/components/PluginDetail.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from "react"
+import {
+  useShowDetailsFor,
+  useGlobalsActions,
+  usePluginConfig,
+} from "./StoreProvider"
+import {
+  CodeBlock,
+  Container,
+  DataGrid,
+  DataGridRow,
+  DataGridCell,
+  DataGridHeadCell,
+  JsonViewer,
+  Panel,
+  Pill,
+  PanelBody,
+  Stack,
+  Tabs,
+  TabList,
+  Tab,
+  TabPanel,
+} from "juno-ui-components"
+
+// Renders the plugin details panel
+const PluginDetail = () => {
+  const pluginConfig = usePluginConfig()
+  const { setShowDetailsFor } = useGlobalsActions()
+  const showDetailsFor = useShowDetailsFor()
+  const [plugin, setPlugin] = useState(null)
+
+  useEffect(() => {
+    if (!showDetailsFor || !pluginConfig) {
+      return
+    }
+    setPlugin(pluginConfig.find((p) => p.id === showDetailsFor))
+  }, [showDetailsFor, pluginConfig])
+
+  const onPanelClose = () => {
+    setShowDetailsFor(null)
+  }
+
+  return (
+    <Panel
+      opened={!!showDetailsFor}
+      onClose={onPanelClose}
+      size="large"
+      heading={plugin?.disabled ? plugin?.name + " DISABLED" : plugin?.name}
+    >
+      <PanelBody>
+        <Tabs>
+          <TabList>
+            <Tab>Details</Tab>
+            <Tab>Raw Data</Tab>
+          </TabList>
+          <TabPanel>
+            <Container px={false} py>
+              <DataGrid
+                columns={2}
+                cellVerticalAlignment="top"
+                gridColumnTemplate="35% auto"
+              >
+                <DataGridRow>
+                  <DataGridHeadCell>Name</DataGridHeadCell>
+                  <DataGridCell>{plugin?.name}</DataGridCell>
+                </DataGridRow>
+
+                {plugin?.disabled && (
+                  <DataGridRow>
+                    <DataGridHeadCell>Disabled</DataGridHeadCell>
+                    <DataGridCell>{plugin?.disabled.toString()}</DataGridCell>
+                  </DataGridRow>
+                )}
+
+                <DataGridRow>
+                  <DataGridHeadCell>Version</DataGridHeadCell>
+                  <DataGridCell>{plugin?.version}</DataGridCell>
+                </DataGridRow>
+
+                <DataGridRow>
+                  <DataGridHeadCell>Cluster</DataGridHeadCell>
+                  <DataGridCell>{plugin?.clusterName}</DataGridCell>
+                </DataGridRow>
+
+                <DataGridRow>
+                  <DataGridHeadCell>External Links</DataGridHeadCell>
+                  <DataGridCell>
+                    {plugin?.externalServicesUrls?.map((url) => {
+                      return (
+                        <a
+                          href={url.url}
+                          target="_blank"
+                          rel="noreferrer"
+                          key={url.url}
+                        >
+                          {url.name}{" "}
```

Comment:
Not a fan of the `{" "}`. If you want to create margin to potential siblings, please use css. E.g.
```jsx
<a
  href=...
  ...
  className="mr-3"
>
```